src/PlusServer/vtkPlusCommandProcessor.cxx

```diff
@@ -38,6 +38,7 @@ See License.txt for details.
 #include "vtkPlusStartStopRecordingCommand.h"
 #include "vtkPlusUpdateTransformCommand.h"
 #include "vtkPlusVersionCommand.h"
+#include "vtkPlusCapistranoCommand.h"

```

Comment:
When building the [PlusBuild](https://github.com/PlusToolkit/PlusBuild) project pointing to this PlusLib branch and WITHOUT Capistrano device support there are linking errors and the project fails to build successfully. This include of `vtkPlusCapistranoCommand.h` likely causes this error because the `vtkPlusCapistranoCommand` class contains  `#include "vtkPlusCapistranoVideoSource.h"`
```
8>  vtkPlusCapistranoCommand.cxx
8>C:\P\PR22\PlusLib\src\PlusServer\Commands\vtkPlusCapistranoCommand.cxx(10,10): fatal error C1083: Cannot open include file: 'vtkPlusCapistranoVideoSource.h': No such file or directory [C:\P\PR22\PlusLib-bin\src\PlusServer\vtkPlusServer.vcxproj]
```